src/Microsoft.NET.Sdk.Functions.Generator/MethodInfoExtensions.cs

```diff
@@ -77,6 +83,41 @@ public static FunctionJsonSchema ToFunctionJson(this MethodDefinition method, st
             };
         }
 
+        /// <summary>
+        /// Gets bindings from return expression used with a binding expression.
+        /// Ex:
+        ///     [FunctionName("HttpTriggerWriteToQueue1")]
+        ///     [return: Queue("myqueue-items-a", Connection = "MyStorageConnStra")]
+        ///     public static string Run([HttpTrigger] HttpRequestMessage request) => "foo";
+        /// </summary>
+        private static JObject[] GetOutputBindingsFromReturnAttribute(MethodDefinition method)
+        {
+            if (method.MethodReturnType == null)
+            {
+                return Array.Empty<JObject>();
+            }
+
+            var outputBindings = new List<JObject>();
+            foreach (var attribute in method.MethodReturnType.CustomAttributes)
+            {
+                if (!attribute.AttributeType.FullName.StartsWith("Microsoft.Azure.WebJobs"))
```

Comment:
code elsewhere is using the [IsWebJobsAttribute](https://github.com/Azure/azure-functions-vs-build-sdk/blob/32ebc25b282fe08ebb283178150450755f2d1044/src/Microsoft.NET.Sdk.Functions.Generator/AttributeExtensions.cs#L50) helper. Won't that work here?

Reply:
Yes. Switched to that. Thank you!